### PR TITLE
refresh clone

### PR DESCRIPTION
**Pull request by Abhishekism9450**
Clone Refresh based on : 
1. snapshot id
2. timestamp

### CODE REVIEW

File: nutanix/resource_nutanix_ndb_clone_refresh.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+	resp, err := conn.Service.RefreshClone(ctx, req, cloneID)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	// Get Operation ID from response of clone refresh and poll for the operation to get completed.
+	opID := resp.Operationid
+	if opID == "" {
+		return diag.Errorf("error: operation ID is an empty string")
```
Comment (bhati-pradeep) [inline nutanix/resource_nutanix_ndb_clone_refresh.go:81]: If there is no operation triggered then do we require this to error out ?

**Review by bhati-pradeep — APPROVED** [review]
LGTM

File: nutanix/resource_nutanix_ndb_clone_refresh.go (hunk 1 above)
Comment (Abhishekism9450) [reply nutanix/resource_nutanix_ndb_clone_refresh.go:81]: Operation is triggered but response doesn't have operation id to poll. so , we throw error saying there is no operation id to poll .

**Review by abhimutant — APPROVED** [review]

**Pull request merged**